app/models/manageiq/providers/kubevirt/inventory/parser.rb

```diff
@@ -189,10 +192,32 @@ def process_domain(domain, uid, name)
     hw_object.cpu_cores_per_socket = domain.cpu&.cores
     hw_object.cpu_total_cores = domain.cpu&.cores
 
-    # Return the created inventory object:
     vm_object
   end
 
+  def process_status(vm_object, status)
+    hw_object = hw_collection.find_or_build(vm_object)
+
+    # Create the inventory object for vm network device
+    ip_address = hardware_networks(hw_object, status)
+    # hardware_guest_devices(hw_object, status, ip_address)
```

Comment:
is it intended to comment line 203?

Reply:
no...leftovers that should have been cleared out.